pom.xml

```diff
@@ -308,24 +308,16 @@
                 </configuration>
             </plugin>
 
-            <!--<plugin>-->
-            <!--<groupId>org.codehaus.mojo</groupId>-->
-            <!--<artifactId>findbugs-maven-plugin</artifactId>-->
-            <!--<version>3.0.1</version>-->
-            <!--<configuration>-->
-            <!--<findbugsXmlOutput>true</findbugsXmlOutput>-->
-            <!--</configuration>-->
-            <!--<executions>-->
-            <!--<execution>-->
-            <!--<id>findbugs</id>-->
-            <!--<phase>test</phase>-->
-            <!--<goals>-->
-            <!--<goal>findbugs</goal>-->
-            <!--</goals>-->
-            <!--</execution>-->
-            <!--</executions>-->
-            <!--</plugin>-->
-
+            <!--
+            The various MINA dependencies rely on artifacts of type 'bundle'
+            so add support to Maven
```

Comment:
Maybe keep a reference to https://issues.apache.org/jira/browse/DIRMINA-919 or some better link that explains what MINA does?

Reply:
Added a comment with a link to some information about OSGi bundles.